code/client/scripts/index.js

```diff
@@ -1175,9 +1175,9 @@ function ShareLink(channel, entity, product_imgurl, product_name, product_brand)
     var url = "", tag_names = "";
 
     if (entity == "product") 
-        url = escape(window.location.href.replace(window.location.pathname, "") + "/app/product/" + g_open_productid);
+        url = encodeURI(window.location.href.replace(window.location.pathname, "") + "/app/product/" + g_open_productid);
```

Comment:
I believe all of these should be `encodeURIComponent()` as they are going to be used in a query string.  `encodeURI` won't encode the `://` correctly and will break the final URL that is constructed.